README.md

```diff
@@ -7,6 +7,49 @@ height="42" />
+* Under Store we have a `Products & Services` submenu option which provides following options: 
+  1. Easy access to Add new product/Import CSV of products into your store. 
+  2. Option to setup bookings & appointments or offer gift cards on purchases via YITH extended plugins.
+* Next submenu under Store is `Sales & Promotions`where customer can setup different promotional offers on products in their store. 
+  1. Here we display 13 different campaign setup options offered by `WonderCart` plugin under `All Campaigns` tab. 
+  2. There is also a `Settings` tab where users can customize look & feel of how sales campaigns are displayed on their Store page.
+* Next submenu under Store is `Payments` where we store owner can setup payments option accepted on purchases at Store. 
+  1. There 3 online payment options via payments gateways like PayPal, Stripe & RazorPay(only for India location). 
+  2. We also offer offline payments option via cheque, bank transfer or cash on delivery.
+* Final submenu under Store is `Store details` with following options:
+  1. Store location and complete address, email address of store owner and currency in which products/services are displayed on the store.
+  2. Under `shipping option` tab on the store details page you can connect to Shipping partner like, Shippo (based on your store location).
+  3. Under `tax settings` tab, store owner can choose to enable/disable sales tax. 
+
+
+## Critical Paths
+
+1. Ecommerce Module should display, `Home` page with site preview, links to Hosting brand a/c and links to pending & completed tasks to setup Ecommerce store.
+2. If currently WordPress site is in coming soon mode, WordPress administrator should see visit site/ store CTA and also a button to Launch or go live under `Ready to go live` section. 
+3. `Store` page should give a brief overview of recent customer activity on store. It also has a section to setup `Ecommerce features` with purchase/enable/manage YITH extended plugins. 
+4. YITH PayPal Payment, WonderCart, YITH Stripe Payments and Ecomdash plugins require WooCommerce to be installed and activated. If WooCommerce isn't setup, `Store` tab should ask customers to first Install & activate WooCommerce. `Home` should display warnings for the same.
+5. `Products & Services` submenu must facilitat CTA's to Add new product/Import CSV of products into store. Also disply services offered by YITH extended plugins.
+6. `Sales & Promotions` submenu allows customers to install `WonderCart` plugin if it is not already installed & activated. Once `WonderCart` is activated it offers option to setup different promotional campaigns.
+7. `Payments` submenu should display payment gateway setup & offline payments setup options.
+8. Stores geographical address, owners email address & currency used on store should be setup under `Store details` tab. Apart from these, option to setup 3rd party shipping setup and enable/disable sales tax should also be set from `Store details` tab. 
```

Comment:
Aren't these just options that show up elsewhere in WooCommerce? If so, I wouldn't consider them a critical path.

Reply:
Customers can setup their `Shippo` account **only** from `Store details` section and Ecommerce module facilitates the connection. Hence keeping `Shippo` setup as critical path.